src/Orchard/Mvc/AntiForgery/AntiForgeryAuthorizationFilter.cs

```diff
@@ -53,91 +75,35 @@ public class AntiForgeryAuthorizationFilter : FilterProvider, IAuthorizationFilt
                        .AntiForgery.Equals("enabled", StringComparison.OrdinalIgnoreCase));
         }
 
-        private static string GetArea(RouteData routeData) {
+        static string GetArea(RouteData routeData) {

```

Comment:
No real reason other than it's not strictly needed. I'll add them back in since it seems far more common practice to explicitly declare private variables and methods.